### ISSUE TITLE
Show in GUI the type of Wallet (HD or old non-HD)

### ISSUE THREAD

**Issue by wodry** [bitcoin/bitcoin#8508]
Using 0.13.0rc1 (or rc2?) I emptied my old non-HD wallet, shutdown bitcoin-core, moved the old wallet.dat away and started bitcoin-core again, to get a new HD wallet created. So the new wallet should be a HD wallet, because I did not change the default "-usehd" option value of 1.

But I would like to see a confirmation of my guess somewhere in the GUI, maybe in the debug window, or maybe under Help->About wallet. Maybe also the wallet should have a version number to be able to clearly identifiy the type of wallet (e.g. wallet type version 1 = HD Wallet)

**Comment by MarcoFalke** [comment]
I think you can try "About"->"Debug window"->"Console" and type in getwalletinfo. Search for walletversion and hdmasterkeyid.

**Comment by wodry** [comment]
Thanks for the tip. Walletversion is 130000 and there is a hdmasterkeyid shown. So I guess all is ok. Maybe a little GUI info would be nice for people who do not bother researching and executing rpc commands

**Comment by luke-jr** [comment]
Sounds reasonable to add the wallet version to the debug console maybe?

**Comment by laanwj** [comment]
> Sounds reasonable to add the wallet version to the debug console maybe?

`getwalletinfo` would be the place for that. I'm surprised it isn't in there yet.
Oh wait it is:
```
obj.push_back(Pair("walletversion", pwalletMain->GetVersion()));
```

**Comment by MarcoFalke** [comment]
Anything left to do here?

**Comment by jonasschnelli** [comment]
Closing, fixed by #8517.
